specs/decisionmatrix/csi.yaml

```diff
@@ -0,0 +1,10 @@
+rows:
+  - instance_type: "*"
+    instance_min_drives: 1
+    instance_max_drives: 12
+    region: "*"
+    min_size: 32
+    max_size: 8192
```

Comment:
we can double the max size to go upto 16 TiB. I just saw the aws and azure yaml files and both go upto 16.

Reply:
aren't we going on max for vshpere?

Reply:
If aws and azure are going upto 16 TiB, no reason to stop at 8 for vsphere. So vsphere should bump up too. But that can be for another day.